internal/cmd/commands/database/migrate.go

```diff
@@ -7,12 +7,14 @@ import (
 	"github.com/hashicorp/boundary/internal/cmd/config"
 	"github.com/hashicorp/boundary/internal/errors"
 	host_plugin_assets "github.com/hashicorp/boundary/plugins/host"
+	kms_plugin_assets "github.com/hashicorp/boundary/plugins/kms"
```

Comment:
Reviewers: Same changes as in base.go. Leaving a comment here mostly for me so that if there were concerns in base.go I remember to address them here too.